proto/ibc/core/channel/v2/query.proto

```diff
@@ -23,7 +23,12 @@ service Query {
 
   // PacketAcknowledgement queries a stored acknowledgement commitment hash.
   rpc PacketAcknowledgement(QueryPacketAcknowledgementRequest) returns (QueryPacketAcknowledgementResponse) {
-    option (google.api.http).get = "/ibc/core/channel/v2/channels/{channel_id}/acknowledgements/{sequence}";
+    option (google.api.http).get = "/ibc/core/channel/v2/channels/{channel_id}/packet_acks/{sequence}";
```

Comment:
changed to `packet_acks`, can make `packet_acknowledgements` if people find it better